### PR TITLE
feature/keyboardShouldPersistTapsUpdate

### PR DESCRIPTION
**Pull request by jeffreymendez1993**
updating keyboardShouldPersistTaps to always since true and false is deprecated.

### CODE REVIEW

File: src/CreditCardInput.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import ReactNative, {
   StyleSheet,
   ScrollView,
   Dimensions,
-  TextInput,
```
Comment (jeffreymendez1993) [inline src/CreditCardInput.js]: TextInput was also unused.

**Comment by hellsingblack** [comment]
@sbycrosz mergeable ?

**Comment by jeffreymendez1993** [comment]
@sbycrosz hey is this safe to merge?

**Comment by afrakhan123** [comment]
Please can you merge as the error keep popping up and is annoyong

**Comment by RigoTheCEO** [comment]
We need to merge this now.

**Comment by jeffreymendez1993** [comment]
Its already on master closing this pr.

**Pull request closed without merging**